www/templates/html/MediaPlayer/v3/Video.tpl.php

```diff
@@ -10,6 +10,16 @@
 }
 
 $autoplay = 'autoplay muted';
+if (isset($controller->options['muted'])) {
+    switch (strtolower($controller->options['muted'])) {
+        // Treat these strings as false and remove muted attribute from $autoplay
+        case '0':
+        case 'false':
+        case '':
```

Comment:
Is this wanted?  If you had _?autoplay&muted_ wouldn't you want 'autoplay muted' instead of 'autoplay'?

Reply:
Well, I guess currently, _?autoplay_ doesn't trigger autoplay right now

Reply:
scratch this- - that's how attributes work muted=muted